{{cookiecutter.project_shortname}}/cookiecutter_templates/AsyncComponent.react.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import { {{cookiecutter.component_name}} as RealComponent } from '../LazyLoader';
```

Comment:
We could add but *not* use the asyncDecorator here (comment out correct usage) like in https://github.com/plotly/dash-table/blob/dev/src/dash-table/dash/DataTable.js#L27 and just leave a comment to use it for components that have side-effects (e.g. prop values calculated from other props or that need to render in order to update/process their props)

For example: https://github.com/plotly/dash-table/blob/dev/src/dash-table/dash/DataTable.js#L27

This is an edge case but not one I'd like component developers to have to find out about. It's not terribly bad either if they do decide to turn it on even if not required.. worst case.